src/Core/File/AnimationData.inl

```diff
@@ -1,22 +1,36 @@
+#include <Core/File/AnimationData.hpp>
+
 #include <Core/Log/Log.hpp>
 
 namespace Ra {
 namespace Asset {
 
 /// NAME
-inline std::string AnimationData::getName() const {
-    return m_name;
+inline void AnimationData::setName( const std::string& name ) {
+    m_name = name;
 }
 
 /// TIME
 inline AnimationTime AnimationData::getTime() const {
     return m_time;
 }
 
+inline AnimationTime& AnimationData::getTime() {
+    return m_time;
+}
+
+inline void AnimationData::setTime( const AnimationTime& time ) {
+    m_time = time;
+}
+
```

Comment:
do you mean we can remove  `inline AnimationTime& AnimationData::getTime()`  ? (or have` inline const AnimationTime& AnimationData::getTime()` )